sunpy/timeseries/sources/eve.py

```diff
@@ -57,8 +57,24 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
+    def plot(self, axes=None, title="EVE/ESP Level1", **kwargs):
```

Comment:
Is it not possible to call the generic time series plot function to get a figure and then update the plot?

Reply:
Why though? I'm a bit confused

Reply:
It might some code that could be useful?

Reply:
So what would the generic time series plot function produce? As of now it just plots the timeseries dataframe and isn't specific to any source right

Reply:
Well if the generic one creates the plot for us, we can call it and then update the figure?

Reply:
Hmm I suppose we could do that as well. Would that be better? I'll re-structure this PR then.

Reply:
I don't suppose, `eve esp` and `lyra` would be exceptions because as I mentioned above, they contain multiple axes so we can't call plot which deals with just one axes. Correct me if I'm wrong, I haven't gone into too much depth.

Reply:
Not sure.

Reply:
Nope, I was wrong, setting `subplot=True` would fix the multiple axes problem. Shall I restructure this then?

Reply:
Sure.